src/data/ACTIONS/layers/patch5.3.ts

```diff
@@ -13,5 +13,13 @@ export const patch530: Layer<ActionRoot> = {
 
 		// MNK 5.3 cooldown changes
 		PERFECT_BALANCE: {cooldown: 90},
+
+		// AST 5.3 changes
```

Comment:
Summary like, `AST 5.3 MP cost changes` or something makes this comment more meaningful.